packages/react-magma-dom/src/components/NavTabs/NavTab.tsx

```diff
@@ -30,6 +30,10 @@ export interface BaseNavTabProps
    * If true, the component will display with the active/selected state
    */
   isActive?: boolean;
+  /**
+   * If true, sets a focus on the specified NavTab
+   */
+  isFocused?: boolean;
```

Comment:
I may be missing something, but I'm wondering why this needs to be prop. `isFocused` is a boolean, not a tab index, so can we by default always focus the first tab, or is there a reason why we shouldn't?

Reply:
This approach was used to satisfy the `cloneElement` so `NavTab` could pass something to `NavTabs`, aside from that, it's always going to focus the first one by default. Should that change?

Reply:
I see. Does this work as we expect if someone passes `isFocused` to the second or third tab child?

Reply:
This works on any NavTab as well indeed.